tests/validator/test_validation_graph.py

```diff
@@ -185,3 +197,279 @@ def test_ExpectationValidationGraph_get_exception_info(
 
     assert left_exception in exception_info
     assert right_exception in exception_info
+
+
+@pytest.mark.integration
+def test_parse_validation_graph():
+    class FakePandasExecutionEngine:
```

Comment:
```suggestion
    class DummyPandasExecutionEngine:
```

I would call it a dummy, not a fake.
"Fakes" are typically full implementations.
https://martinfowler.com/articles/mocksArentStubs.html#TheDifferenceBetweenMocksAndStubs

>Dummy objects are passed around but never actually used. Usually they are just used to fill parameter lists.

>Fake objects actually have working implementations, but usually take some shortcut which makes them not suitable for production (an [in memory database](https://martinfowler.com/bliki/InMemoryTestDatabase.html) is a good example).

Reply:
@Kilo59 I called this one "Fake" because it does a little bit more than a Dummy in the sense that it sets the type of the object.  Do you still think that it should be "Dummy"?  Thanks!

Reply:
I see, on second look this, might be more of a "Stub"

>Stubs provide canned answers to calls made during the test, usually not responding at all to anything outside what's programmed in for the test.

`PandasExecutionEngineStub`

Reply:
@Kilo59 Thank you very much -- updated!